File: ARCHIVE/uri/1-Beginner/1094.c
#include <stdio.h>

struct animal{
	int val;
	float per;
};

int main(){

	int i;
	char id;
	struct animal c = {0, 0}, r = {0, 0}, s = {0, 0};
	int qnt, temp, total;
	scanf("%d", &qnt);

	for (i = 0; i < qnt; i++){

		scanf("%d %c", &temp, &id);
		switch (id){
			case 'C':
				c.val += temp;
				break;
			case 'R':
				r.val += temp;
				break;
			case 'S':
				s.val += temp;
				break;
		}
	}
	
	total = c.val + r.val + s.val;
	c.per = (float)c.val * 100 / (float)total;
	r.per = (float)r.val * 100 / (float)total;
	s.per = (float)s.val * 100 / (float)total;

	printf("Total: %d cobaias\n", total);
	printf("Total de coelhos: %d\n", c.val);
	printf("Total de ratos: %d\n", r.val);
	printf("Total de sapos: %d\n", s.val);
	printf("Percentual de coelhos: %0.2f %\n", c.per);
	printf("Percentual de ratos: %0.2f %\n", r.per);
	printf("Percentual de sapos: %0.2f %\n", s.per);

	return 0;
}

File: ARCHIVE/2018-03-28-2100/1094.py
n = input ()

coelho = 0
rato = 0
sapo = 0
for i in range (0, n):
    line = raw_input ()
    line = line.split (' ')
    numb = float(line[0])
    specie = line[1]
    if (specie == 'C'):
        coelho+=numb
    elif (specie == 'R'):
        rato+=numb
    elif (specie == 'S'):
        sapo+=numb
total = float (coelho + rato + sapo)
print "Total: " + str(int(total)) + " cobaias"
print "Total de coelhos: " + str (int(coelho))
print "Total de ratos: " + str (int(rato))
print "Total de sapos: " + str (int(sapo))
print "Percentual de coelhos: " + "%.2lf" % round (coelho*100/total, 2) + " %"
print "Percentual de ratos: " + "%.2lf" % round (rato*100/total, 2) + " %"
print "Percentual de sapos: " + "%.2lf" % round (sapo*100/total, 2) + " %"
